src/components/BookItem.js

```diff
@@ -1,21 +1,22 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import { useDispatch } from 'react-redux';
-import { removeBook } from '../redux/books/books';
+import { removeBookAsync } from '../redux/books/books';
 import './reset.css';
 import styles from './BookItem.module.css';
 
 function BookItem({ book }) {
   const dispatch = useDispatch();
   const removeHandler = () => {
-    dispatch(removeBook(book.id));
+    dispatch(removeBookAsync(book.item_id));
   };
 
   return (
     <div>
     <div className={styles.bookItem}>
-      <p className={styles.itemCategory}>Action</p>
+      <p className={styles.itemCategory}>{book.category}</p>
       <h2 className={styles.itemTitle}>{book.title}</h2>
-      <p className={styles.itemAuthor}>{book.author}</p>
+      {/* <p className={styles.itemAuthor}>{book.author}</p> */}
```

Comment:
- [ ] Kindly delete all commented out code. This helps in writing clean and readable code